e2e/run-e2e-suite.sh

```diff
@@ -93,8 +95,52 @@ until kubectl get secret | grep -q ^external-secrets-e2e-token; do \
   sleep 3; \
 done
 
-echo -e "${BGREEN}Starting external-secrets e2e tests...${NC}"
+echo -e "${BGREEN}Starting external-secrets e2e tests with the custom resource manager disabled...${NC}"
+kubectl rollout status deploy/localstack
+kubectl rollout status deploy/e2e-kubernetes-external-secrets
+
+kubectl run \
+  --attach \
+  --restart=Never \
+  --env="LOCALSTACK=true" \
+  --env="LOCALSTACK_SSM_URL=http://ssm" \
+  --env="LOCALSTACK_SM_URL=http://secretsmanager" \
+  --env="AWS_ACCESS_KEY_ID=foobar" \
+  --env="AWS_SECRET_ACCESS_KEY=foobar" \
+  --env="AWS_DEFAULT_REGION=us-east-1" \
+  --env="AWS_REGION=us-east-1" \
+  --env="LOCALSTACK_STS_URL=http://sts" \
+  --env="DISABLE_CUSTOM_RESOURCE_MANAGER=true" \
+  --generator=run-pod/v1 \
+  --overrides='{ "apiVersion": "v1", "spec":{"serviceAccountName": "external-secrets-e2e"}}' \
+  e2e --image=external-secrets-e2e:test
+
+echo "Preparing for second round of e2e tests..."
```

Comment:
This isn't great for E2E testing the custom resource manager disabled/enabled.

I think instead adding flags/env vars to this script and using a build matrix in the workflow file such as:
```yaml
  test-e2e:
    runs-on: ubuntu-latest
    name: E2E
    strategy:
      matrix:
        customResourceManagerDisabled: [true, false]
```

would be cleaner. However, builds will use more resources until the 4.x release where the custom resource manager is removed. What do you think? @Flydiverny @silasbw 

It also begs the question as to whether or not Helm v2 should be tested. The build matrix approach could also be leveraged there to test all combinations of custom resource manager settings and Helm versions.

Though `ubuntu-latest` [does not come with Helm v2](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md) so there would need to be a conditional step for installing Helm v2 in the E2E job.